scripts/microbe_stage/microbe_editor/microbe_editor.as

```diff
@@ -563,6 +576,28 @@ class MicrobeEditor{
     }
 
 
+    bool checkIsNucleusPresent()
+    {
+        double lengthMicrobe = double(editedMicrobe.length());
```

Comment:
Why is there an unused variable here?

~~Why is this a double???? the length is of type `uint` and not a floating point type, mixing them is not good.~~